modules/apps/contacts/contacts-web/src/main/resources/META-INF/resources/js/main.js

```diff
@@ -327,26 +324,24 @@ AUI.add(
 				_editEntry(contact) {
 					var instance = this;
 
-					var config = instance._config;
-
-					var portletURL = new Liferay.PortletURL.createURL(
-						config.baseRenderURL
-					);
+					var editEntryRenderParameters = {
+						entryId: contact.entryId,
+						'javax.portlet.action': 'deleteEntry',
```

Comment:
We are declaring edit entry render to be a delete entry action, I'm assuming this is a copy/paste leftover. Render still works as this param ends up being ignored.